Microsoft.Toolkit.Services/Services/MicrosoftGraph/MicrosoftGraphAuthenticationHelper.cs

```diff
@@ -190,10 +191,21 @@ public bool Logout()
         /// </summary>
         /// <returns>Success or failure</returns>
         public bool LogoutV2()
```

Comment:
Can you mark this method as Obsolete so we can inform people to use the new method?
```suggestion
        [Obsolete("This method will be removed, please use LogoutAsync instead.")]
        public bool LogoutV2()
```

We should do this on the `Logout` method above as well.